pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
+DJANGO_SETTINGS_MODULE=oneanddone.settings
+addopts = --reuse-db
+sensitive_url=mozilla\.org
```

Comment:
Curious to find out why not `setup.cfg`? Do we have one for flake8? We should include that in the Travis setup too.